Products/CMFPlone/PasswordResetTool.py

```diff
@@ -134,33 +137,26 @@ def resetPassword(self, userid, randomstring, password):
         # clean out the request
         del self._requests[randomstring]
         self._p_changed = 1
-    ## Implementation ##
-
-    # external
-
-    security.declareProtected(ManagePortal, 'setExpirationTimeout')
 
+    @security.protected(ManagePortal)
     def setExpirationTimeout(self, timedelta):
         """Set the length of time a reset request will be valid in days.
         """
         self._timedelta = abs(timedelta)
 
-    security.declarePublic('getExpirationTimeout')
-
+    @security.public
     def getExpirationTimeout(self):
         """Get the length of time a reset request will be valid.
         """
         return self._timedelta
 
-    security.declarePublic('checkUser')
-
+    @security.declarePublic
```

Comment:
Oh yes, this is by mistake and probably the reason why the tests are failing. Thanks for the hint.